src/core/index.ts

```diff
@@ -1,17 +1,25 @@
 import { authedFetch, cyrb53, uuidv4 } from './utils';
 import { del, get, set } from './storage';
 
-const SDK_VERSION = '1.2.1';
+const SDK_VERSION = '1.2.2';
```

Comment:
This is the only 'real' change in this file, rest is formatting from updating Prettier.